docs/development/README.md

```diff
@@ -74,6 +75,29 @@ chmod +x .git/hooks/pre-commit
 _Please note that the Github workflow will still prevent this from merging
 unless the CHANGELOG is updated._
 
+### Pocket Network CLI
+
+Pocket provides a CLI for interacting with Pocket Core. The CLI is meant to be an user but also a machine friendly way for interacting with Pocket Network.
+
+The commands available are listed [here](../../app/client/cli/doc/commands/client.md).
+
+In order to build the CLI:
+
+1. Generate local files
+
+```bash
+make develop_start
+```
+
+2. Build the CLI
+
+```bash
+make build
```

Comment:
Do you know why I'm hitting an issue?
![Screenshot 2022-12-27 at 1 53 00 PM](https://user-images.githubusercontent.com/1892194/209710034-63b656bc-7092-47f9-b39d-19a93c730f83.png)

Reply:
Do you have the same output when issuing `make build-pocket` ?

Reply:
Yes, I do. Confirmed.

Reply:
Yes, I do. Confirmed.

Reply:
@Gustavobelfort Still hitting this issue.

I haven't had time to dive deep into debugging this so what do you suggest I try?

See this video as a reference: https://drive.google.com/file/d/14GzCAhnESWFz2a_h2twuZZcovcj8mPL4/view?usp=sharing

I ran `make develop_test` to show that it worked, followed by `make build-pocket`.

I anticipate others will hit the same issue if I do too.

Reply:
Can you provide me the output of:
`go env`
and
`env`
?

Reply:
![Screenshot 2023-01-03 at 1 46 08 PM](https://user-images.githubusercontent.com/1892194/210421533-6b6c456a-b257-4743-82ee-92fb1ce83b0a.png)

Which parts of `env` do you need?

Reply:
Any of the following if they're set: `GOTAGS`, `LDFLAGS`, `BUILD_DIR`, `GOARGS`

Reply:
Also, can you try building the `main.go` file in the `docgen` directory manually ?

Reply:
I'm guessing that your problem is your `LDFLAGS`, they look malformed :)

Can you try:
`unset LDFLAGS` and then `make build` ?

Reply:
Seems like I had flags set up on my machine to make `readline` work on a mac following the suggestions here: https://groups.google.com/g/sage-devel/c/cZj2QsUIwL4

It is likely a local setup issues, but if it's common, I'm sure we'll find out sooner rather than later.